src/util/kaldi-thread.h

```diff
@@ -260,6 +268,8 @@ class TaskSequencer {
     args->me->tot_threads_avail_.Signal();
   }
 
+  int num_threads_; // copy of config.num_threads (since Semaphore doesn't store original count)
```

Comment:
thanks.
please make this int32, just to forestall compiler warnings on some systems and compilers.f